recipes/fruitpizza.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fruit Pizza</title>
+</head>
+<body>
+    <h1>Fruit Pizza</h1>
+    <img src="https://pinchofyum.com/wp-content/uploads/Fruit-Pizza-Design-1.jpg" width="350" height="450" alt="a colorful fruit pizza on a serving plate">
```

Comment:
A good habit to start getting into is breaking up attributes on multiple lines. Our current style guide at work makes mention that 3 or more attributes constitute the need to break into separate lines. 

```
<img width="350"
      height="450"
      src="https://pinchofyum.com/wp-content/uploads/Fruit-Pizza-Design-1.jpg"
      alt="a colorful fruit pizza on a serving plate">
```